UnofficialCrusaderPatch/Version.cs

```diff
@@ -400,6 +400,44 @@ public class Version
                 }
             },
             
+            new Change("o_fix_small_wall_placement_count", ChangeType.Bugfix, true)
+            {
+                new DefaultHeader("o_fix_small_wall_placement_count")
+                {
+
+                    new BinaryEdit("o_fix_small_wall_placement_count_pre") // 437FB1
+                    {
+                        new BinAddress("CurrentlyControlledPlayer", 2),
```

Comment:
CurrentlyControlledPlayer = 01A275DC
instead: 01A275D8 (player of the task being processed)